connect-wallet/README.md

```diff
@@ -0,0 +1,39 @@
+# Connect Wallet in the Browser
+
+This example sets up a new React app with Metaplex using Next.js.
+
+This tutorial shows how to connect your wallet to the browser via [Solana Wallet Adapter Library](https://github.com/solana-labs/wallet-adapter), and use the [Metaplex JS SDK](https://github.com/metaplex-foundation/js) to display a random NFT from the wallet and refresh it when a button is clicked.
+
+This example has been generated using the following steps:
+
+1. **Create a new project using Next.js.**
+    Redirect to [this](https://github.com/metaplex-foundation/js-examples/tree/main/getting-started-nextjs) link to know how to create next.js application and install metaplex successfully.
+
+2. **Install the needed dependencies.**
+
+   ```sh
+    npm install @solana/wallet-adapter-base \
+         @solana/wallet-adapter-react \
+         @solana/wallet-adapter-react-ui \
+         @solana/wallet-adapter-wallets
+   ```
+
+3. **Create the script files and copy the code from the corresponding file in the repository**
+   ```
+      cd pages
+      touch ShowNFTs.js MetaplexProvider.js useMetaplex.js
+   ```
```

Comment:
Could you please split that step into three steps, to explain what each file does in more detail? It would be great to have some code snippets here as well so the reader can quickly grasp what they need to do.